food_ratings/frontend/views.py

```diff
@@ -56,7 +56,6 @@ def _entry(register, key):
     response = _get('%s/record/%s.json' % (url, key))
     return response.json()
 
-
```

Comment:
although now I think about it I think two blank lines is [PEP8 style](https://www.python.org/dev/peps/pep-0008/#blank-lines) 😆